### ISSUE TITLE
Integration with trust tokens?

### ISSUE THREAD

**Issue by jkarlin** [csharrison/aggregate-reporting-api#9]
To help reduce fraudulent aggregate reports, it seems like it would be useful to be able to require a trust token redemption in order to send an aggregate report for a document. This requirement would likely need to be expressed in the document response headers.

**Comment by csharrison** [comment]
Yes, I am working on an update to this explainer with authentication built in. I think we can do something even more useful than raw trust tokens, so stay tuned.
